kodilla-good-patterns/src/main/java/com/kodilla/flightservice/FlightService.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.flightservice;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FlightService {
+    private FlightsList flightsList = new FlightsList();
+
+
+    public List<String> allCitiesFrom(String city){
+        return flightsList.getFlightsList().stream()
```

Comment:
klasa flightsList ma zmienna flightsLists. To mylace. Nazwij klase np. FlightsRepository